website/docs/authenticating_via_azure_cli.html.markdown

```diff
@@ -9,13 +9,13 @@ description: |-
 
 # Authenticating to Azure Resource Manager using the Azure CLI
 
-Terraform supports authenticating to Azure through a couple of different means - via a Service Principal and using the Azure CLI.
+Terraform supports authenticating to Azure through two different means; a Service Principal or the Azure CLI.
```

Comment:
🤔 reading this again, I'm wondering if this might read better as `...through the Azure CLI or a Service Principal (Shared Account)` - what do you think?

Reply:
Can do!